libs/containers/Webhook.lua

```diff
@@ -85,4 +89,110 @@ function get.defaultAvatarURL(self)
 	return self:getDefaultAvatarURL()
 end
 
+
+-- Setters and Modifiers
+function Webhook:_modify(payload)
+	local data, err = self.client._api:modifyWebhook(self._id, payload)
+	if data then
+		self:_load(data)
+		return true
+	else
+		return false, err
+	end
+end
+
+function Webhook:setAvatar(avatar)
+	self._avatar = avatar and Resolver.base64(avatar) or json.null
+
+	local data, err = self:_modify({avatar = self._avatar})
+	if data then
+		return true
+	else
+		return nil, err
+	end
+end
+
+function Webhook:setChannelId(channelId)
+	local data, err = self:_modify({channel_id = channelId})
+	if data then
+		return true
+	else
+		return nil, err
+	end
+end
+
+function Webhook:setName(name)
+	local data, err = self:_modify({name = name})
+	if data then
+		return true
+	else
+		return nil, err
+	end
+end
+
+-- Rest of functions
+function Webhook:send(content, options) -- return message
```

Comment:
Why is this an options table instead of a content table? This should also probably not be named "send".